SPL/SPL_c1.php

```diff
@@ -779,7 +779,7 @@ public function fscanf($format, &...$vars) {}
          * the end of <i>string</i> is reached, whichever comes
          * first.
          * </p>
-         * @return int the number of bytes written, or 0 on error.
+         * @return int|bool the number of bytes written, or false on error.
          */
```

Comment:
@martin-juul Please use #[LanguageLevelTypeAware] attribute to set return types different for different PHP versions. You can find examples of usages of attribute across stubs.